SETUP.md

```diff
@@ -42,7 +41,7 @@ We install the dependencies with Conda. As a pre-requisite, we want to make sure
 
 ```{shell}
 conda update conda -n root
-conda update anaconda
```

Comment:
about this @roalexan had problems in windows, we were planning to remove these two lines:

```
conda update conda -n root
conda update anaconda 
```
and then add a maximum conda version to the conda file generator. Robert do you have the info about this?

Reply:
Adding to conda file generator sounds a simple & good solution.
For windows, I didn't have any issues to install the environments except `conda update anaconda` which threw the package not installed error. (I ran the script on Administrator-terminal)

Reply:
Btw, please remove this part once we have updated conda file generator.